docs/release-notes/RELEASE-NOTES-0.12.0.md

```diff
@@ -23,11 +23,13 @@ The instructions below describe:
 
 ## Retrieve the software for running a node
 
-You can run a node by using either:
-* The published [release binary][this-release] corresponding to your platform; or
-* The published [Docker image `aeternity/epoch`][docker].
+You can run a node by either:
+* Install the published [release binary][this-release] corresponding to your platform; or
+* Run the published [Docker image `aeternity/epoch`][docker]; or
+* [Build it from source][build].
 
 [docker]: https://github.com/aeternity/epoch/blob/v0.12.0/docs/docker.md
+[build]: https://github.com/aeternity/epoch/blob/v0.12.0/docs/build.md
```

Comment:
As this requires sources anyway, this doc build.md does not need to be included in release binary. Ok.